includes/transifex-live-integration-defaults.php

```diff
@@ -120,6 +121,7 @@ static function settings() {
 				'rate_us' => 'https://wordpress.org/support/view/plugin-reviews/transifex-live-integration?rate=5#postform',
 				'api_key_landing_page' => 'https://app.transifex.com/signup/?utm_source=liveplugin',
 				'translate_urls' => 'https://help.transifex.com/en/articles/6259014-technical-instructions#h_e9a258ad3b',
+        'canonical_urls' => 'https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls'
```

Comment:
Is it acceptable that we point to google docs and not ours? (Maybe it is I am not sure)

Reply:
As per discussion with Aggeliki, let's remove for now and we can re-iterate later if needed.

Reply:
Removed!